_install-and-configure/install-opensearch/rpm.md

```diff
@@ -57,36 +70,23 @@ This guide assumes that you are comfortable working from the Linux command line
    # Install the arm64 package using rpm.
    sudo rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
    ```
-For OpenSearch 2.12 and greater, a custom admin password is required in order to set up a security demo configuration.  To set a custom admin password, use one the following commands:
-   ```bash
-    # Install the x64 package using yum.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-
-   # Install the x64 package using rpm.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-
-   # Install the arm64 package using yum.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-
-   # Install the arm64 package using rpm.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
 1. After the installation succeeds, enable OpenSearch as a service.
-    ```bash
-    sudo systemctl enable opensearch
-    ```
-    {% include copy.html %}
+   ```bash
+   sudo systemctl enable opensearch
+   ```
+   {% include copy.html %}
 
 1. Start OpenSearch.
-    ```bash
-    sudo systemctl start opensearch
-    ```
-    {% include copy.html %}
+   ```bash
+   sudo systemctl start opensearch
+   ```
+   {% include copy.html %}
 
 1. Verify that OpenSearch launched correctly.
```

Comment:
Beyond the line break, what status should users expect OpenSearch to return when running this command?

```suggestion
1. Verify that OpenSearch launched correctly:

```

Reply:
If users run `curl -I localhost:9200` and get `HTTP/1.1 200 OK` in the response, this should confirm all is well.